projects/Array-Rotation/README.md

```diff
@@ -0,0 +1,62 @@
+---
+title: List/Array rotation in Python
+layout: default
+date: 2020-10-14
+last-modified: 2020-10-14
+featured-image: {FEATURED IMAGE NAME}
```

Comment:
```suggestion
featured-image: 
```

If you want, you can put an image here. Just make sure to add one to assets/sources.